abmarl/sim/gridworld/examples/gridworld_example.py

```diff
@@ -7,83 +7,62 @@
     AttackingAgent, HealthAgent
 from abmarl.sim.gridworld.state import PositionState, HealthState
 from abmarl.sim.gridworld.actor import MoveActor, AttackActor
-from abmarl.sim.gridworld.observer import GridObserver
+from abmarl.sim.gridworld.observer import SingleGridObserver
 from abmarl.tools.matplotlib_utils import mscatter
 
 
 class WallAgent(GridWorldAgent):
     """
-    Wall agents, immobile and view blocking.
-
-    Args:
-        encoding: Default encoding is 1.
-        render_shape: Default render_shape set to 'X'.
+    Wall agents, immobile, and view blocking.
     """
-    def __init__(self, encoding=1, render_shape='X', **kwargs):
+    def __init__(self, **kwargs):
+        kwargs['view_blocking'] = True
```

Comment:
We don't need to add it to kwargs, we can just pass it to the super init as its own kwarg